### ISSUE TITLE
update to 0.7.0 show empty page

### ISSUE THREAD

**Issue by DragonHeart69** [henrygd/beszel#247]
After updating to version 0.7.0, only a blank page is loaded. I tried starting with an empty data folder, but this gives the same problem. In the logs I see nothing special.
```
2024/11/03 12:21:53 Systems synced with config.yml

2024/11/03 12:21:53 Server started at http://0.0.0.0:8090

├─ REST API: http://0.0.0.0:8090/api/

└─ Admin UI: http://0.0.0.0:8090/_/
```

**Comment by dkorecko** [comment]
Same issue here, in Chrome Dev tools console, getting this:
```
Error loading sk Error: Unknown variable dynamic import: ../locales/sk/sk.ts
    at index-Ddqw86lM.js:50:69416
    at new Promise (<anonymous>)
    at $P (index-Ddqw86lM.js:50:69322)
    at Qc (index-Ddqw86lM.js:50:76944)
    at index-Ddqw86lM.js:50:78373
```

![image](https://github.com/user-attachments/assets/8630b629-7ab0-4fef-b205-f1e2c72480a7)

**Comment by matsuricorp** [comment]
Same issue for me. Tried restoring a backup and ran the update again with the same result.

**Comment by henrygd** [comment]
Sorry about that! Just released a patch in 0.7.1 that should fix it.

Let me know if it still happens.
